elytron/src/main/java/org/wildfly/extension/elytron/ElytronDefinition.java

```diff
@@ -294,6 +294,10 @@ public void registerChildren(ManagementResourceRegistration resourceRegistration
         resourceRegistration.registerSubModel(SSLDefinitions.getServerSNISSLContextDefinition());
         resourceRegistration.registerSubModel(new CertificateAuthorityDefinition());
         resourceRegistration.registerSubModel(new CertificateAuthorityAccountDefinition());
+        ResourceDefinition dynamicClientSSLContextDefinition = SSLDefinitions.getDynamicClientSSLContextDefinition(false);
+        if (resourceRegistration.enables(dynamicClientSSLContextDefinition)) {
+            resourceRegistration.registerSubModel(dynamicClientSSLContextDefinition);
+        }
```

Comment:
This is redundant.  The MRR already filters registration based on stability (do you don't have to).

Reply:
Fixed!